src/ImageRunModal.jsx

```diff
@@ -328,104 +331,128 @@ export class ImageRunModal extends React.Component {
+                                </FormSelect>
+                            </InputGroup>
+                        </FormGroup>
+
+                        { this.state.image.isSystem &&
+                            <FormGroup fieldId='run-image-cpu-priority' label={_("CPU shares")}>
+                                <InputGroup className="ct-input-group-spacer-sm modal-run-limiter" id="run-image-dialog-cpu-priority">
+                                    <Checkbox id="run-image-dialog-cpu-priority-checkbox"
+                                        className="pf-u-align-content-center"
+                                        isChecked={this.state.cpuSharesConfigure}
+                                        onChange={checked => this.onValueChanged('cpuSharesConfigure', checked)} />
+                                    <TextInput type='number'
+                                        id="run-image-cpu-priority"
+                                        value={dialogValues.cpuShares}
+                                        step={1}
+                                        min={2}
+                                        max={262144}
+                                        isReadOnly={!this.state.cpuSharesConfigure}
+                                        onChange={value => this.onValueChanged('cpuShares', value === "" ? "" : parseInt(value))} />
+                                </InputGroup>
+                            </FormGroup>}
+
+                        <FormGroup fieldId='run-image-dialog-start-after-creation' label={_("Start after creation")} hasNoPaddingTop>
+                            <Checkbox isChecked={this.state.runImage} id="run-image-dialog-start-after-creation" onChange={value => this.onValueChanged('runImage', value)} />
+                        </FormGroup>
+                    </Tab>
+                    <Tab eventKey={1} title={<TabTitleText>{_("Integration")}</TabTitleText>} id="create-image-dialog-tab-integration" className="pf-l-grid pf-m-gutter">
 
-                <FormGroup fieldId='run-image-dialog-publish' label={_("Ports")}>
-                    <DynamicListForm id='run-image-dialog-publish'
+                        <FormGroup fieldId='run-image-dialog-publish' label={_("Ports")}>
+                            <DynamicListForm id='run-image-dialog-publish'
                                      formclass='publish-port-form'
                                      onChange={value => this.onValueChanged('publish', value)}
                                      default={{ IP: null, containerPort: null, hostPort: null, protocol: 'tcp' }}
                                      itemcomponent={ <PublishPort />} />
-                </FormGroup>
+                        </FormGroup>
 
-                <FormGroup fieldId='run-image-dialog-env' label={_("Volumes")}>
-                    <DynamicListForm id='run-image-dialog-volume'
+                        <FormGroup fieldId='run-image-dialog-env' label={_("Volumes")}>
```

Comment:
This fieldId is wrong . I know you just moved it - but if you repush wanna add a fix up commit?

Reply:
Fixed